src/pages/conference/info/conferenceInfo/InfoForm.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+import { reduxForm, Field } from 'redux-form';
+
+import RaisedButton from 'material-ui/RaisedButton';
+import Subheader from 'material-ui/Subheader';
```

Comment:
Why won't both 
```
import RaisedButton from 'material-ui/RaisedButton';
import Subheader from 'material-ui/Subheader';
```
be wrote like
```
import { Subheader, RaisedButton } from 'material-ui';
```

Reply:
Because abusolute import will reduce the size of the project :)